pkg/kube/pod_test.go

```diff
@@ -1244,3 +1245,18 @@ func (s *PodSuite) TestAddAnnotations(c *check.C) {
 		c.Assert(tc.podOptions, check.DeepEquals, tc.expectedPodOptions)
 	}
 }
+
+func (s *PodSuite) TestSomething(c *check.C) {
```

Comment:
I think we should either rename this method or try to add comment that explains what exactly we are planning to test in this.

Reply:
great catch, thank you !
I've renamed it and made it working as I wanted originally.